### PR TITLE
Configure bumpversion, create changelog and include contributing guidelines

### PR DESCRIPTION
**Pull request by victor-torres**

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -0,0 +1,78 @@
+Contributing
+============
+
+``web-po`` is an open-source project and we'd love to accept your
```
Comment (kmike) [inline CONTRIBUTING.rst:4]:
```suggestion
``web-poet`` is an open-source project and we'd love to accept your
```

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+* Fork the project
+* Create a new branch with your patch
+* Open a pull request to master
+* Ask for code reviews from other contributors
+
+Ideally, tests should keep passing and coverage should not decrease after a
+commit.
+
+How to release a new version
```
Comment (kmike) [inline CONTRIBUTING.rst:30]: This information is not for contributors, it is for maintainers.

Comment (victor-torres) [reply CONTRIBUTING.rst:30]: move to MAINTAINING.rst?

Comment (kmike) [reply CONTRIBUTING.rst:30]: yeah, or to wiki.

Comment (victor-torres) [reply CONTRIBUTING.rst:30]: moved to a new file

**Comment by kmike** [comment]
Thanks @victor-torres!

**Pull request merged**